docs/en_US/install/3ds.md

```diff
@@ -28,8 +39,46 @@ We make use of 3 Luma patches to connect your console to Pretendo:
 2) `0004013000003202` - Friends system module. This patch replaces the `https://nasc.nintendowifi.net` URL with our servers URL 
 3) `0004013000003802` - act system module. This patch replaces the `https://account.nintendo.net/v1/api/` URL with our servers URL
 
-## How does it work?
+<div class="tip">
+ℹ️ <b>Skip this step if Luma patches are already enabled on your console.</b>
+</div>
+
+In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the SELECT button on your 3DS and power it on.
```

Comment:
Button prompts should be made more clear. Since we use markdown, just using `BUTTON` is enough

```suggestion
In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the `SELECT` button on your 3DS and power it on.
```